zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -234,6 +238,18 @@ public void hostProgress(int id, String hostAndPort, String msg, int percentage)
         }
     }
 
+    @Override
+    public void filteredMessage(HttpMessage msg, String reason) {
+        try {
+            filterMessageTableModel.addScanResult(SessionStructure.getNodeName(msg), reason);
```

Comment:
It can use the URI from the message instead of node name.

Reply:
so can i use msg.getRequestHeader().getURI().toString() or shall i again call SessionStructure.getHostName(msg) ?

Reply:
I believe he meant `msg.getRequestHeader().getURI().toString()`

Reply:
sure thanks.

Reply:
Right.